build.sbt

```diff
@@ -50,6 +50,7 @@ lazy val cassandraBundle = project
     libraryDependencies += ("org.apache.cassandra" % "cassandra-all" % "3.11.3")
         .exclude("commons-logging", "commons-logging"),
     dependencyOverrides += "com.github.jbellis" % "jamm" % "0.3.3", // See jamm comment in https://issues.apache.org/jira/browse/CASSANDRA-9608
+    dependencyOverrides += "net.java.dev.jna" % "jna" % "5.12.1",
```

Comment:
Reading a comment here: https://issues.apache.org/jira/browse/CASSANDRA-16212?focusedCommentId=17230445&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17230445

I'm a bit cautious bumping it like this, since it seems it could break the launcher on other platforms.

Reply:
I understand. I've put together a step-by-step guide for working around this issue on M1 Macs here: https://github.com/awwsmm/apc-mwe

The TL;DR is that you can generate your own `cassandra-bundle.jar` by running the `cassandraBundle/assembly` task in `akka-persistence-cassandra`, then use that jar in third-party applications that depend on `akka-persistence-cassandra`.

Reply:
Thanks for writing that up! 👍